cmd/nerdctl/commit.go

```diff
@@ -39,6 +39,7 @@ func newCommitCommand() *cobra.Command {
 	}
 	commitCommand.Flags().StringP("author", "a", "", `Author (e.g., "nerdctl contributor <nerdctl-dev@example.com>")`)
 	commitCommand.Flags().StringP("message", "m", "", "Commit message")
+	commitCommand.Flags().StringSlice("override-cmd", nil, "Override image CMD")
```

Comment:
This should be like `--change='CMD ["apachectl", "-DFOREGROUND"]'`
https://docs.docker.com/engine/reference/commandline/commit/